Source/WebKit/Configurations/_WebKit_SwiftUI.xcconfig

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2024 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+// OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#include "BaseTarget.xcconfig"
+
+// InstallAPI is transitively enabled through including BaseTarget.xcconfig.
+
+PRODUCT_NAME = _WebKit_SwiftUI; // Cross-import overlay projects must have this exact naming scheme.
+PRODUCT_BUNDLE_IDENTIFIER = com.apple.WebKit.$(PRODUCT_NAME:rfc1034identifier);
+INFOPLIST_FILE = Info.plist;
+
+INSTALL_PATH = $(WEBKIT_FRAMEWORKS_DIR);
```

Comment:
If this is going to be installed next to WebKit.framework in WEBKIT_FRAMEWORKS_DIR, you'll need to add a "Create Symlink to Alt Root Path" script phase like WebKit.framework does. That adds the symlink in /System/Library/Frameworks to the real path in the OS cryptex, which allows clients to load the framework from the SDK.

(As an aside, I've been thinking about going in and renaming all the "alternate root" stuff to more explicitly refer to the OS cryptex by name, since we don't need the extra obfuscation for secrecy anymore, and that just is asking to be confused with things like WK_ALTERNATE_FRAMEWORKS_DIR, which refers to Mac Catalyst. Maybe one of these days.)